src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2489,6 +2489,7 @@ codet java_bytecode_convert_methodt::convert_instructions(
           INVARIANT(
             a_it2->second.stack.size() == stack.size(),
             "Stack sizes should be the same.");
+          stackt::const_iterator os_it=a_it2->second.stack.begin();
```

Comment:
This shouldn't be in the formatting commit

Reply:
This introduces a variable the last commit deleted and should be squashed into the appropriate commit.

Reply:
I have now moved this fix into the correct commit.